pkgs/development/python-modules/annexremote/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, future
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "annexremote";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
```

Comment:
Please add a comment with the reason for not using `fetchPypi`.

Reply:
See 48a5d481baccb2d49305ed863b45c57f92ac6cd0